nilearn/plotting/img_plotting.py

```diff
@@ -834,8 +837,9 @@ def plot_stat_map(stat_map_img, bg_img=MNI152TEMPLATE, cut_coords=None,
             symmetrical.
         symmetric_cbar: boolean or 'auto', optional, default 'auto'
             Specifies whether the colorbar should range from -vmax to vmax
-            or from 0 to vmax. Setting to 'auto' will select the latter if
-            the whole image is non-negative.
+            or from vmin to vmax. Setting to 'auto' will select the latter if
```

Comment:
vmin is a tiny bit misleading given that it you can not pass vmin to these functions. I am not sure how to rephrase it better though, so we should probably leave that for later.